cloudify_cli/tests/resources/blueprints/local/blueprint_without_plugins.yaml

```diff
@@ -0,0 +1,12 @@
+tosca_definitions_version: cloudify_dsl_1_0
```

Comment:
Make sure indentation is 2.
Perhaps "cloudify_dsl_1_1" should be used here.
Too many blank lines at the end of the file :)